multiversx_sdk_cli/cli_deps.py

```diff
@@ -34,16 +35,38 @@ def install(args: Any):
 
 def check(args: Any):
     name: str = args.name
-    module = dependencies.get_module_by_key(name)
-    tag_to_check: str = config.get_dependency_tag(module.key)
+
+    if name == "all":
+        all_dependencies = dependencies.get_all_deps()
+        missing_dependencies: List[Tuple[str, str]] = []
+
+        for dependency in all_dependencies:
+            tag_to_check: str = config.get_dependency_tag(dependency.key)
+            is_installed = check_module_is_installed(dependency, tag_to_check)
+
+            if not is_installed:
+                missing_dependencies.append((dependency.key, tag_to_check))
+
+        if len(missing_dependencies):
+            raise errors.DependenciesMissing(missing_dependencies)
+        return
+    else:
+        module = dependencies.get_module_by_key(name)
+        tag_to_check: str = config.get_dependency_tag(module.key)
+
+        is_installed = check_module_is_installed(module, tag_to_check)
+        if is_installed and name != "rust":
```

Comment:
For Rust, we don't display this log info? Or without this `if` the message will be duplicated by the one from `modules.py`? I think better to have it duplicated on the CLI than to have this custom `if` here (e.g. in the future we will forget its reason).

Reply:
Without the `if` statement the message will be duplicated by the one in `modules.py`. Why I added it it's because if you have a different Rust version other than the recommended one, you'll see a warning that the versions don't match but the log will show that the correct version is installed since it takes the tag from the config.
A problem that could occur with the other dependencies as well, now that I think about it, but is less common.